app/repos/persons.js

```diff
@@ -99,6 +100,33 @@ const getPersons = async (queryParams, options = {}, transaction) => {
   }
 }
 
+const deletePersons = async (personsToDelete, user) => {
+  const result = {
+    count: {
+      failed: 0,
+      success: 0
+    },
+    deleted: {
+      failed: [],
+      success: []
+    }
+  }
+
+  for (const personReference of personsToDelete) {
```

Comment:
Do we need to do the deletes in one transaction i.e. all or nothing? Would we have a problem if only some got deleted?